pages/en/posts/dgapi-2-migration.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Digital Goods API 1.0 to 2.0 Migration Guide'
+metadesc: Digital Goods API 2.0 is coming in M96. Make sure you update your app to ensure it continues to work.
+tags:
+  - product news
+  - web
+authors:
+  - ikarahan
+date: 2021-10-01
+---
+
+We are making some changes to the Digital Goods API (DGAPI) that will address feedback we’ve heard from you and other developers, as well as close some potential fraud avenues. DGAPI 2.0 will require some changes in your web app and backend, but you won’t need to upload a new package to the Play Store.
+
+The new API will land in Chrome M96 whose stable launch is scheduled for November 15th on Android, and November 30th on Chrome OS.
+
+Please note that some of your users might be still using Chrome M95 or earlier even after M96 is available, hence your app must work with both DGAPI 1.0 and 2.0 until all users upgrade to M96+.
+
+This guide will take you through the steps to update your web app and backend in order to make it compatible with both DGAPI 1.0 and 2.0. In order to avoid breakage, please make your changes before the Chrome M96 launch date.
```

Comment:
```suggestion
This guide will take you through the steps to update your web app and backend in order to make it compatible with both DGAPI 1.0 and 2.0. In order to avoid breakage, please make your changes before the Chrome 96 launch date.
```